src/application-status-overlay/ApplicationStatusOverlayProvider.jsx

```diff
@@ -6,6 +6,9 @@ import StatusView from 'terra-status-view';
 import ThemeContext from 'terra-theme-context';
 import ApplicationStatusOverlayContext from './ApplicationStatusOverlayContext';
 import styles from './ApplicationStatusOverlayProvider.module.scss';
+import 'mutationobserver-shim';
```

Comment:
With inert being initialized automatically with the ApplicationBase import, these imports (and the `require` on line 79) no longer need to be included here.

Reply:
Let's go ahead and remove it from this file since it's not needed.

Reply:
Removed 235f298